get-started/part3.md

```diff
@@ -132,7 +132,13 @@ docker stack deploy -c docker-compose.yml getstartedlab
 Our single service stack is running 5 container instances of our deployed image
 on one host. Let's investigate.
 
-Get the service ID for the one service in our application:
+To get the services for a service stack, get the service ID for the one service in our application:
+```shell
+docker stack services getstartedlab
```

Comment:
You mention;

> When you use `docker service ls`, it won't show you the list of services. Instead use `docker stack services getstartedlab` to show the service that was created and thereby getting the Service ID of it.

I don't think that's the case; after deploying the stack, the service should be visible;

```
docker service ls

ID                  NAME                MODE                REPLICAS            IMAGE               PORTS
bqpve1djnk0x        getstartedlab_web   replicated          5/5                 nginx:alpine        *:4000->80/tcp
uhq4zdcedw64        test                replicated          1/1                 nginx:alpine       
```

However, as can be seen in my example output; other services (not related to the stack you deployed) may be visible as well.

I think the current documentation is correct (and it shows that a service that's deployed through a stack also can be found using the regular `docker service ls` command). Using `docker stack services` can be useful though, as it makes it easier to find the services _just_ for a particular stack.

Perhaps instead, we can add a paragraph after the "Look for output for the `web` service" paragraph to show `docker stack services` as an (easier) alternative? Something like;

    Look for output for the `web` service, prepended with your app name. If you
    named it the same as shown in this example, the name is
    `getstartedlab_web`. The service ID is listed as well, along with the number of
    replicas, image name, and exposed ports.

    You can also use the `docker stack services` command, followed by the name of
    your stack. Run the following command to view all the services that are part
    of the `getstartedlab` stack:

    ```bash
    docker stack services getstartedlab
    ID                  NAME                MODE                REPLICAS            IMAGE               PORTS
    bqpve1djnk0x        getstartedlab_web   replicated          5/5                 username/repo:tag   *:4000->80/tcp
    ```


There's also `docker stack ps` (to show all tasks of a stack), but not sure if we should add that as well (open for input though);

```bash
docker stack ps getstartedlab
ID                  NAME                  IMAGE               NODE                DESIRED STATE       CURRENT STATE           ERROR               PORTS
uwiaw67sc0eh        getstartedlab_web.1   username/repo:tag   docker-desktop      Running             Running 9 minutes ago                       
sk50xbhmcae7        getstartedlab_web.2   username/repo:tag   docker-desktop      Running             Running 9 minutes ago                       
c4uuw5i6h02j        getstartedlab_web.3   username/repo:tag   docker-desktop      Running             Running 9 minutes ago                       
0dyb70ixu25s        getstartedlab_web.4   username/repo:tag   docker-desktop      Running             Running 9 minutes ago                       
aocrb88ap8b0        getstartedlab_web.5   username/repo:tag   docker-desktop      Running             Running 9 minutes ago        
```

Reply:
Sure, that sounds good to me. Please do add the paragraph having `docker stack services` explained.

Reply:
@thaJeztah I'm all for showing all possiible example commands that will be helpful. Let me add the content you've suggested.